genesiscommunitysuccess/blank-app-seed
# chore: bumped packages version , added path in design-system-configurator script FUI-1052

📓 &nbsp; **Related JIRA**

[FUI-1052](https://genesisglobal.atlassian.net/browse/FUI-1052)

🤔 &nbsp; **What does this PR do?**

It bumps packages to latest version so that the app created from blank app seed have latest features available including latest changes to design system configurator app. It also adds a path param for  design system configurator app script that allows copying and using design tokens from the file at the specified path provided as a param. 

- update the versions of genesislcap packages to latest 14.12.4
- update the `configure-design-system` script in client/web./package.json to include path param
- fix for node version upgrade

🚀 &nbsp; **Where should the reviewer start?**
Check updated versions and script in client/web/package.json

📑 &nbsp; **How should this be tested?**
- run `npm run bootstrap` in client directory
- run `npm run configure-design-system` in client/web director
- run `npm run dev` in client/web
Once you have both of apps running. go to Design System configurator app. Change Accent, Neutral color etc and Click on Save. Then Click Save again. Later Go to Client app and refresh to view your changes. You can also check the changes you made in client/web/src/styles/design-tokens.json
[blank-app-seed-dsc-changes.webm](https://user-images.githubusercontent.com/118179643/236264836-3d82d307-8fab-4560-95bc-ace232c5a061.webm)

✅ &nbsp; **Checklist**
